doc-content/enterprise-only/cases/case-management-creating-manager-approval-proc.adoc

```diff
@@ -2,88 +2,92 @@
-
-*2*: The *Object Library*.
-
-*3*: The *Properties* panel displays the properties of the selected element. If no element is selected, the panel contains process properties.
-
 .Procedure
-. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* -> *IT_Orders_New* -> *orderhardware Business Process (legacy)*.
-. In the process designer, expand *Tasks* in the *Object Library* drag a new *User* task on to the canvas.
-. Click the new user, click image:cases/left-arrows.png[] to open the *Properties* panel, and input the following values:
-* *Groups* - `supplier`
-* *Name* - `Prepare hardware spec`
-* *Task Name* - `PrepareHardwareSpec`
-* *Description* - `Prepare hardware specification for #{initiator} (order number #{CaseId})`
-
-. Click the down arrow in the *Assignments* field, add the following new conditions in the *Prepare hardware spec Data I/O* window. For the first output assignment, select *Custom* for the *Data Type* and input `org.jbpm.document.Document`. Click *Save*.
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* -> *IT_Orders_New* -> *orderhardware Business Processes*.
+. Create and configure the *Prepare hardware spec* user task:
+.. Drag a start event on to the canvas and create an outgoing connection from the start event to a task and convert the new task to a user task.
+.. Click the new user task, and in the upper-right corner, click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. Input `Prepare hardware spec` in the *Name* field.
+.. Expand *Implementation/Execution*, click *Add* below the *Groups* menu, click *Select* -> *New*, and input `supplier`.
+.. Input `PrepareHardwareSpec` in the *Task Name* field.
+.. Select the *Skippable* check box and enter the following in the *Description* field:
 +
-image::cases/hardware-spec-data.png[Prepare hardware I/O values]
-
-. Click the *Prepare hardware spec* user task and create a new user task.
+`Prepare hardware specification for #{initiator} (order number #{CaseId})`
+.. Click image:getting-started/diagram_properties.png[] in the *Assignments* field and add the following new conditions in the *Prepare hardware spec Data I/O* window.
 +
-image::cases/new-user-task.png[Creating a new user task]
-
-. Click the new user task, click image:cases/left-arrows.png[] to open the *Properties* panel, and input the following values:
-* *Actors* - `manager`
-* *Name* - `Manager approval`
-* *Task Name* - `ManagerApproval`
-* *Description* - `Approval request for new hardware for #{initiator} (order number #{CaseId})`
+image::cases/hardware-spec-data.png[Prepare hardware I/O values]
 
-. Click the down arrow in the *Assignments* field, add the following new conditions in the *ManagerApproval Data I/O* window. For the first output assignment, select *Custom* for the *Data Type* and input `org.jbpm.document.Document`. Click *Save*.
+.. Click *Save*.
+. Create and configure the *Manager approval* user task:
```

Comment:
I don't see why this is bolded. Only bold if you are directly referring to a UI element. For example, click *Manager approval*. I also wonder why the cap M?

Reply:
Makes sense. Changed to lower case - no bold